AlpheusUnitTests/ExecuteCommandTests.fs

```diff
@@ -8,3 +8,40 @@ open ItisLab.Alpheus.Tests.Utils
 open ItisLab.Alpheus
 open ItisLab.Alpheus.ExecuteCommand
 
+[<Fact>]
```

Comment:
Just wonder why some of the facts are within a class and this one is outside?

Reply:
@dvoits , That is because some tests does not require unique file environment (thus declared as simple functions) while other require setting up and clearing of single use directory (implemented via deriving from SingleUseDir type)